apps/api/src/app/activity/usecases/get-activity-graph-states/get-activity-graph-states.usecase.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@nestjs/common';
 import { MessageRepository } from '@novu/dal';
```

Comment:
this file's directory is being renamed in addition to the file itself being renamed